universal-did/src/main/java/me/uport/sdk/universaldid/UniversalDID.kt

```diff
@@ -27,6 +27,13 @@ object UniversalDID : DIDResolver {
         resolvers[resolver.method] = resolver
     }
 
+    /**
+     * Check to see if any resolvers are registered
+     */
+    fun hasResolver(): Boolean {
```

Comment:
A better approach would be to use the already existing `canResolve()` method.

Here are some blank DIDs of known type that can be tested to see if a resolver is registered:
```
val blankUportDID = "did:uport:2nQs23uc3UN6BBPqGHpbudDxBkeDRn553BB"
val blankEthrDID = "did:ethr:0x0000000000000000000000000000000000000000"
val blankHttpsDID = "did:https:example.com"
```